setup.py

```diff
@@ -76,12 +78,15 @@ def get_version():
 
         if dirty != "":
             version += ".dev1"
-
     else:
-        # Extract the version from the PKG-INFO file.
-        with open(join(d, "PKG-INFO")) as f:
-            version = version_re.search(f.read()).group(1)
-
+        try:
```

Comment:
Interesting.  Lint was complaining about the open not being in a try block but not about the fact that it has no encoding specified?  I am surprised.